config/kubernetes/sig-scheduling/teams.yaml

```diff
@@ -61,6 +61,15 @@ teams:
     - kerthcet
     - sanposhiho
     privacy: closed
+  sig-scheduling-approvers:
```

Comment:
For discussion: Can we only keep three teams for clearness, like `sig-scheduling-leads`, `sig-scheduling-approvers` and `sig-scheduling-misc` since most of the team members are the same.

Reply:
+1 unless other teams are created by a rule in the community (since other sigs also have xxx-bugs, xxx-misc etc, there _may_ be some rules about teams? not sure.)

Reply:
> For discussion: Can we only keep three teams for clearness, like `sig-scheduling-leads`, `sig-scheduling-approvers` and `sig-scheduling-misc` since most of the team members are the same.

SGTM. @sanposhiho could you update this PR by de-duping redundant aliases? (raising a separate one is also fine, but maybe not worth it)

Reply:
👍 Done.

Reply:
/lgtm